peerloop/domain/user/dtos.py

```diff
@@ -13,12 +10,7 @@ class RegisterRequest(BaseModel):
 
     @validator("email")
     def email_must_be_valid(cls, v: str) -> str:
-        try:
-            validation = validate_email(v)
-            email = validation.email
-        except EmailNotValidError:
-            raise InvalidEmailError(f"Invalid Email Error: {v} is not a valid email address.")
-        return email
+        return validate_email_format(v)
```

Comment:
If we make the function to return a boolean, then we can put condition statement and return the email if that's true.

Reply:
Thanks for your suggestion! I think it is more appropriate way to separate email validation logic.

Commit: [682b0a6](https://github.com/peerloop/peerloop-backend/pull/5/commits/682b0a6d44e32483bc46c5a23601ceb9a6416e83)